src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -92,8 +92,8 @@
     // studyId -> study EKID
     private final Map<UUID, UUID> studies = new HashMap<>();
 
-    private final Map<String, String> userAppsDict                  = Collections.synchronizedMap( new HashMap<>() );
-    private final Set<UUID>           notificationEnabledStudyEKIDs = new HashSet<>();
+    private final Set<String> systemUserAppPackageNames     = Collections.synchronizedSet( new HashSet<>() );
```

Comment:
Nitpick on the naming... in my mind, there's system apps and there's user apps, so "system user apps" doesn't make sense to me. I recommend dropping the `User` part and renaming to `systemAppPackageNames` or `androidSystemAppPackageNames` or something like that.